README.adoc

```diff
@@ -1,34 +1,35 @@
-# SignalFx Integrations
+= SignalFx Integrations
```

Comment:
Why are  we switching to "adoc"  format?  I'm pretty sure this isn't going to work with the existing scripting unless you have confirmed that it does.

Reply:
I didn't switch anything. All I did is run `to-product-docs` on `signalfx-org-metrics`. It looked like the only way to move my changes to `product-docs`.

Reply:
Your changes will get automatically pushed to product-docs via the jenkins integrations that push automatically when you merge here. Don't do anything to run to-product-docs manually please. Follow these instructions if you need to preview your changes in the doc site (it is a hacky workaround): https://signalfuse.atlassian.net/wiki/spaces/DOCS/pages/1068892423/Access+and+update+integrations+repo#Option-2:-Test-your-changes-to-the-documentation-site